WcaOnRails/app/views/competitions/_index_competitions_list.html.erb

```diff
@@ -7,30 +7,55 @@
     <% if @past_selected %>
       <div class="col-md-12" id="past-comps">
         <% table_title = year_is_a_number?(params[:year]) ? t('competitions.index.titles.past', year: params[:year]) : t('competitions.index.titles.past_all') %>
-        <%= render 'index_table', competitions: competitions, title: table_title %>
```

Comment:
Doesn't the old `index_table` need deleting?

Reply:
yes!